spec/mailers/previews/admin_mailer_preview.rb

```diff
@@ -0,0 +1,9 @@
+# Preview all emails at http://localhost:3000/rails/mailers/admin_mailer
```

Comment:
Is this comment here needed?

Reply:
This was autogenerated by `rails generate`, I thought it would be nice to keep it.

But thinking about it now, it may be better to include this information in the README.md or in the setup docs instead.